src/components/search/stats.tsx

```diff
@@ -1,28 +1,23 @@
 import React from 'react'
-import {connectStats} from 'react-instantsearch-dom'
+import {useStats, StatsProps} from 'react-instantsearch'
 
-type CustomStatsProps = {
-  nbHits: number
-  searchQuery: string
-}
+const Stats: React.FunctionComponent<React.PropsWithChildren<StatsProps>> = (
+  props,
+) => {
+  const {nbHits, query} = useStats(props)
```

Comment:
The code mod script initially created custom connect functions that use the hooks API.

I decided to instead integrate the hooks pattern into the components.